### DISCUSSION TITLE
Clarify the meaning of extra_money_earned_on_exports entity

### DISCUSSION THREAD

**Discussion by FraBoCH** [hif2k1/battery_sim#97] [Q&A]
I’m not sure to understand what this entity represents, and why it’s value is negative ? 
Does it measure the money I’m not earning because I store it instead of reselling it ?
Thanks for clarifying and maybe add a bit of documentation on how to interpret the data collected by the integration.

**Comment by hif2k1** [comment]
Yes, I couldn't think of a good name for this. For some people if you use a
battery you will export less electricity because you use more of your own
generation and therefore not earn as much money for exports. Others who
have a flexible tariff use the battery to import at a cheaper time and
export at an expensive one they can actually earn more money for their
exports. This sensor needs to cover both these options.

In your case you are exporting less than without the simulated battery and
therefore you won't get as much money for your exports as you would without
the battery. Hence why it is reading negative. Hopefully this is much less
what you are saving on imports so it saves you money overall. The total
money saved is simply the sum of these two and is the number to look at
to decide if the battery is worth it.

I did wonder about making it the other way round (i.e. calling the sensor
"money lost on exports"), but I thought it was better for money lost to be
negative. Otherwise there is a double negative which is even more
confusing. However, if you can think of a better name or better way of
explaining please let me know or edit the readme.

Many thanks,

Hamish
